### ISSUE TITLE
BUG: cirrus-ci nightly upload token is no longer valid - nightly builds are not being uploaded from cirrus

### ISSUE THREAD

**Issue by andyfaff** [scipy/scipy#17192]
### Describe your issue.

I was browsing the nightly wheels on https://anaconda.org/scipy-wheels-nightly/scipy/files and noticed that the `macosx_arm64` and `manylinux2014_aarch64` wheels were out of date - they should've been built last weekend.
When I look at the CI build log for the cron job the upload process failed.

The settings for the cron job can be found at https://cirrus-ci.com/settings/repository/4894635377033216. The build log for the last cron job can also be found there, https://cirrus-ci.com/build/5940032849575936. The tail end of the log is below. 

The token is specified in `ci\cirrus_wheels.yml` using an encrypted variable, SCIPY_NIGHTLY_UPLOAD_TOKEN (originally created by @rgommers as SCIPY_NIGHTLY_UPLOAD_TOKEN_CIRRUS). I am not sure why the "Authorization token is no longer valid".

Assigning Ralf because he can create the tokens, and possibly try out a local upload. I wonder if the staging token is also broken.

### Reproducing Code Example

```python
N/A
```

### Error message

```shell
Uploading file "scipy-wheels-nightly/scipy/1.10.0.dev0/scipy-1.10.0.dev0-cp310-cp310-macosx_12_0_arm64.whl"
[WARNING] Distribution "scipy-1.10.0.dev0-cp310-cp310-macosx_12_0_arm64.whl" already exists. Removing.
[ERROR] ('Authorization token is no longer valid', 401)
```

### SciPy/NumPy/Python version information

N/A

**Comment by rgommers** [comment]
> The token is specified in `ci\cirrus_wheels.yml` using an encrypted variable, SCIPY_NIGHTLY_UPLOAD_TOKEN (originally created by @rgommers as SCIPY_NIGHTLY_UPLOAD_TOKEN_CIRRUS). I am not sure why the "Authorization token is no longer valid".

This is very strange, the tokens are still there and valid for a long time into the future. 

Let me xref gh-17000, which is where I did the cleanup a month ago.

**Comment by rgommers** [comment]
xref where we created the tokens (explains also how this actually works): https://github.com/scipy/scipy/pull/17029#issuecomment-1253139344

**Comment by rgommers** [comment]
The Anaconda UI says that `SCIPY_NIGHTLY_UPLOAD_TOKEN_CIRRUS` was never used, so I revoked it. And I generated a new encrypted variable for `SCIPY_NIGHTLY_UPLOAD_TOKEN`:
```
ENCRYPTED[377be83afdaf9e8fa8bac105022bb1d362f72fff8501c2b5e827018b82270111828fb542fd7bd56bbfac9745603bd535]
```
And same for the staging token:
```
ENCRYPTED[5940af5e589adf1647fb2832be2c912812de22b5b6551d93e177600da69b2103ffdf0ee1cf16671cf5e76f69a966bc30]
```

Those encrypted values did change, and I am unclear on why. Maybe there was a change on the CirrusCI side in the encryption/decryption algo.

The token itself works fine, so I'll open a PR.
